src/main/java/se/knowit/bookitregistration/dto/RegistrationDTO.java

```diff
@@ -0,0 +1,11 @@
+package se.knowit.bookitregistration.dto;
+
+import lombok.Data;
+
+@Data
+public class RegistrationDTO {
+
+    private String eventId;
+    private String registrationId;
+    private String email;
```

Comment:
Can we use a `Participant` object here instead with an email as a property/field? That makes it more readable and more straight forward for clients to handle :)

Reply:
Later, in other branch